site/content/contribute/containers/_index.md

```diff
@@ -0,0 +1,52 @@
+
+- [mattermost/mattermost-prod-db](https://hub.docker.com/r/mattermost/mattermost-prod-db) - Community driven image for Database to run together with **mattermost/mattermost-prod-app**. **This Docker repository will be deprecated in Mattermost 6.0**. For more information and to check the dockerfile please refer to https://github.com/mattermost/mattermost-docker
+
+- [mattermost/mattermost-prod-web](https://hub.docker.com/r/mattermost/mattermost-prod-web) - Community driven image for WebServer to run together with **mattermost/mattermost-prod-app**. **This Docker repository will be deprecated in Mattermost 6.0**. For more information and to check the dockerfile please refer to https://github.com/mattermost/mattermost-docker
+
+- [mattermost/mattermost-preview](https://hub.docker.com/r/mattermost/mattermost-preview) - This is a Docker image to install Mattermost in Preview Mode for exploring product functionality on a single machine using Docker. Preview image Docs: http://bit.ly/1W76riY. GitHub: https://github.com/mattermost/mattermost-docker-preview
+
+- [mattermost/platform](https://hub.docker.com/r/mattermost/platform) - Mirror of **mattermost/mattermost-preview**. This is a Docker image to install Mattermost in Preview Mode for exploring product functionality on a single machine using Docker. Preview image (mirror). Docs: http://bit.ly/1W76riY. GitHub: https://github.com/mattermost/mattermost-docker-preview
+
+- [mattermost/mattermost-loadtest](https://hub.docker.com/r/mattermost/mattermost-loadtest) - Image for the Load Test application. Tools for profiling Mattermost under heavy load. GitHub: https://github.com/mattermost/mattermost-load-test
+
+- [mattermost/sync-helpwanted-tickets](https://hub.docker.com/r/mattermost/sync-helpwanted-tickets) - For Internal usage. This iamge runs the sync with Jira tickets and GitHub Issues. To check the code please refer to https://github.com/mattermost/mattermost-utilities/tree/master/github_jira_tools
+
+- [mattermost/mattermost-build-server](https://hub.docker.com/r/mattermost/mattermost-build-server) - Image used to build Mattermost used in CI. To check the Docker file refer to https://github.com/mattermost/mattermost-server/blob/master/build/Dockerfile.buildenv
+
+- [mattermost/mattermost-build-webapp](https://hub.docker.com/r/mattermost/mattermost-build-webapp) - Image used to build Mattermost Webapp used in CI. To check the Docker file refer to https://github.com/mattermost/mattermost-webapp/blob/master/build/Dockerfile
+
+- [mattermost/mattermost-marketplace-build](https://hub.docker.com/r/mattermost/mattermost-marketplace-build) - Image used to build Mattermost Marketplace. For more information please refer https://github.com/mattermost/mattermost-marketplace
```

Comment:
I'm not sure if this image is used any longer. The laste update happend in September. @lieut-data Can you confirm?

Reply:
if not being used anymore i can delete that, which will be better :D

Reply:
Yes, I think we can delete it -- no dependencies within the marketplace.

Reply:
ok, will remove, thanks!

Reply:
done